operations/helm/charts/grafana-agent/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ internal API changes are not present.
 Unreleased
 ----------
 
+### Enhancements
+- Add hostPort specification to extraPorts and extraPort documentation. (@pnathan)
```

Comment:
Can you move this under the existing enhancements header instead of creating a new header?

Reply:
Whoops, sorry about that. Done.